docs/enhancement-proposals/off-ep-0003.md

```diff
@@ -0,0 +1,86 @@
+# OFF-EP 0003 — Allow interpolation of non-integer bond orders
```

Comment:
Suggest title instead be "Allow fractional bond order interpolation between arbitrary fractional bond order interpolation points", since this proposal is really suggesting we enable more elaborate interpolation schemes.